### ISSUE TITLE
Select - keyboard navigation with `search`

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#6096]
### Expected Behavior
You should be able to navigate to the search field using the keyboard

### Actual Behavior
Once you enter the list of options there isn't a way to get back up to the search field without closing the Select.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/input-select-search--search

### Steps to Reproduce

1. Go to https://storybook.grommet.io/?path=/story/input-select-search--search
2. Open the Select
3. Navigate with the keyboard down to "option 1"
4. Try to get back to the search field using the keyboard

### Your Environment

- Grommet version: latest
- Browser Name and version: chrome
- Operating System and version (desktop or mobile):
